src/v3/src/util/passwordUtils.ts

```diff
@@ -49,6 +50,11 @@ const minSymbolValidator = (password: string, limit: unknown): boolean => (
   !(limit as number) || (password.match(/["!#$%&'()*+,-./\\:;<=>?@[\]^_`{|}~]/)?.length || 0) >= (limit as number)
 );
 
+const maxConsecutiveRepeatingCharactersValidator = (password: string, limit: unknown): boolean => {
+  const regex = new RegExp(`(.)\\1{${limit}}`);
+  return !(limit as number) || (Boolean(password.length) && !password.match(regex));
```

Comment:
I know that `Number.isInteger` is not widely available (e.g. not in IE11) but **for Gen3** we have an automatic polyfill generated by `@babel/preset-env` so when you add this it will automatically add a polyfill for it in the production build.

Reply:
will update, thank you